tenon_key_pages.txt

```diff
@@ -0,0 +1,2 @@
+/
```

Comment:
first line will be dropped after demonstrating a failed build, as it's not passing yet

Reply:
sample failure: https://ci--alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fjournal/detail/PR-740/4/pipeline